docs/book/src/SUMMARY.md

```diff
@@ -27,6 +27,7 @@
     - [VM Identity](./topics/vm-identity.md)
     - [Windows](./topics/windows.md)
     - [SSH Access to nodes](./topics/ssh-access.md)
+    - [Consuming Existing Azure Infrastructure](./topics/consuming-existing-azure-infrastructure.md)
```

Comment:
nit: the list of features is in alphabetical order (mostly), can you please more this up? Maybe at E for "Existing Azure Infrastructure" ?